pkg/network/go/goid/goid_offset.go

```diff
@@ -1,5 +1,8 @@
 // Code generated by go generate; DO NOT EDIT.
 
+//go:build linux_bpf
+// +build linux_bpf
```

Comment:
Why does the output need these build tags? We don't use ebpf for the goid offset.

Reply:
`pkg/network/http/gotls/lookup/luts.go` having the build tag dependency already commited in the repo (generated gotls/lookup/main => generate_luts.go) 

but yeah may we should avoid tag pollution